kafka-broker/generic_packer_template.json.j2

```diff
@@ -27,7 +27,7 @@
       "ami_users": "{{ event['ami_users'] }}",
       {% endif %}
       "region": "{{ event['region'] }}"
-    }]{% if 'provision_script_keys' in event %},
+    }]{% if 'provision_script_keys' in event and len(event[provision_script_keys]) > 0 %},
```

Comment:
Sorry again Ian; there was a typo in my example code!  You need quotes around 'provision_script_keys'.